draft-id-claim-169-v1-1-0.md

```
@@ -0,0 +1,375 @@
+
+Claim 169 represents a JSON Object that includes the below table as ID attributes. You can find an illustration of the ID structure contained within Claim 169, where:
+
+## 3. Semantics
+
+### 3.1 CBOR Map Structure Overview
+
+**Note**:
+
+* All the fields here are optional.
+* The issuer of **IDClaim169** is expected to host the **JWKS** file at the standard **.well-known** URL. This allows relying parties to verify the signature of the issued IDClaim169.
+* Please ensure to review the [Guidelines ](169-qr-code-specification.md#guidelines)and important note(s) below with respect to [standard CWT attributes](https://docs.mosip.io/1.2.0/readme/standards-and-specifications/mosip-standards/169-qr-code-specification#note-on-standard-cwt-attributes) and [credential status](https://docs.mosip.io/1.2.0/readme/standards-and-specifications/mosip-standards/169-qr-code-specification#note-on-status-of-credential-cwt).
```

markdownlint-cli2 (0.18.1)

Spaces inside link text

(MD039, no-space-in-links)